common/host_connections.go

```diff
@@ -0,0 +1,37 @@
+package common
+
+import (
+	"fmt"
+	"github.com/frightenedmonkey/statsproxy/config"
+	"log"
+	"net"
+)
+
+var (
+	HostConns            = make(map[config.Host]*net.UDPConn, len(config.Service.Statsd.Hosts))
+	HostChans            = make(map[config.Host]chan string, len(config.Service.Statsd.Hosts))
+	PacketCountPerServer = make(map[config.Host]*uint32)
+	PacketCount          uint32
+)
+
+func InitializeStatsdConns() {
+	for _, host := range config.Service.Statsd.Hosts {
+		addr, err := net.ResolveUDPAddr("udp", host.String())
+
+		if err != nil {
+			log.Fatalf("Failed to resolve host %v - %s", host, err)
+		}
+
+		Logger.Info(fmt.Sprintf("The host address is %v", addr.String()))
+
+		conn, err := net.DialUDP("udp", nil, addr)
+		if err != nil {
+			log.Fatalf("Failed to connect to host %v - %s", addr, err)
```

Comment:
yeah, gotta get the hosts that are listed because otherwise we lose consistent hashing of metrics.